PaddleCV/PaddleDetection/ppdet/utils/checkpoint.py

```diff
@@ -59,13 +61,27 @@ def load_pretrain(exe, prog, path):
         raise ValueError("Model pretrain path {} does not "
                          "exists.".format(path))
 
-    logger.info('Loading pretrained model from {}...'.format(path))
+    is_finetune = len(ignore_params) > 0
+    logger.info('Loading parameters from {}...'.format(path))
+    if is_finetune:
+        logger.info(
+            'Finetune mode and weights related to num_classes are ignored')
 
     def _if_exist(var):
-        b = os.path.exists(os.path.join(path, var.name))
-        if b:
+        do_ignore = False
+        param_exist = os.path.exists(os.path.join(path, var.name))
+        if is_finetune:
+            # Parameter related to num_classes will be ignored in finetuning
+            do_ignore_list = [name in var.name for name in ignore_params]
+            do_ignore = any(do_ignore_list)
+            if do_ignore and param_exist:
+                logger.info(
+                    'When finetuning for {} architecture, ignore {}'.format(
+                        ignore_params[do_ignore_list.index(True)], var.name))
```

Comment:
ignore_params[do_ignore_list.index(True)]  not architecture？

Reply:
Done, thanks

Reply:
LGTM